database/models/reports.py

```diff
@@ -240,3 +243,46 @@ class CompareComponent(MixinBaseClass, CodecovBaseModel):
     patch_totals = Column(postgresql.JSON)
 
     commit_comparison = relationship(CompareCommit, foreign_keys=[commit_comparison_id])
+
+
+class Test(CodecovBaseModel, MixinBaseClass):
+    __tablename__ = "reports_test"
+    id_ = Column("id", types.Text, primary_key=True)
+    external_id = Column(
+        UUID(as_uuid=True), default=uuid.uuid4, unique=True, nullable=False
+    )
+    created_at = Column(types.DateTime(timezone=True), default=get_utc_now)
+    updated_at = Column(
+        types.DateTime(timezone=True), onupdate=get_utc_now, default=get_utc_now
+    )
+
+    @property
+    def id(self):
+        return self.id_
```

Comment:
this is just because we're replacing the `MixinBaseClass` properties, I'm not sure why but `MixinBaseClass` does this for the `id` field and i just wanted to keep it consistent